apps/ledger-live-mobile/src/navigation/useDeepLinking.ts

```diff
@@ -5,7 +5,7 @@ import { filterPlatformApps } from "@ledgerhq/live-common/platform/filters";
 import { getPlatformVersion } from "@ledgerhq/live-common/lib/platform/version";
 import { NavigatorName, ScreenName } from "../const";
 
```

Comment:
actually im unsure it will work with Linking.openURL :thinking: thats why we made a seperate handler in the first place.
to try again later and remove it